changelog.md

```diff
@@ -1,3 +1,5 @@
+2025-10-15 - b87652aaf206421f6e690e84652d7f820bb05d8f - components/src/core/components/Input/Select/SelectInput.vue - Enhanced dropdown component to auto-scroll to the selected option and Add a new prop called scrollToNextOf to auto-select the next option based on a reference value. 
```

Comment:
Update the description now that the implementation has changed. add correct file changes (excluding tests)